### PR TITLE
Update Ember to 3.21

### PR DESCRIPTION
**Pull request by jrjohnson**

### CODE REVIEW

File: addon/classes/resolve-async-value.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { tracked } from '@glimmer/tracking';
+import { Resource } from 'ember-could-get-used-to-this';
+export default  class AsyncProcessResource extends Resource {
```
Comment (stopfstedt) [inline addon/classes/resolve-async-value.js:3]: class name should be `ResolveAsync`. 
remove double blank space before `class`.

File: addon/classes/async-process.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { tracked } from '@glimmer/tracking';
+import { Resource } from 'ember-could-get-used-to-this';
+export default  class AsyncProcessResource extends Resource {
```
Comment (stopfstedt) [inline addon/classes/async-process.js:3]: remove double blank space before `class`.

File: README.md [hunk 3]
```diff
@@ -5,9 +5,12 @@
 - [Ilios Frontend](https://github.com/ilios/common)
 - [Ilios LTI App](https://github.com/ilios/lti-app)
 
-## Usage
+Compatibility
+------------------------------------------------------------------------------
 
-`ember install ilios-common`
+* Ember.js v3.16 or above
```
Comment (stopfstedt) [inline README.md:11]: :tada:

File: addon/components/ilios-calendar.hbs [hunk 4]
```diff
@@ -40,7 +40,7 @@
       {{t "general.loadingEvents"}}
       ...
     </h1>
-    {{#let (concat "ilios-calendar-" @selectedView) as |Calendar|}}
+    {{#let (component (concat "ilios-calendar-" @selectedView)) as |Calendar|}}
```
Comment (stopfstedt) [inline addon/components/ilios-calendar.hbs:43]: nice :+1:

**Review by stopfstedt — CHANGES_REQUESTED** [review]
needs just a few minor changes, otherwise this LGTM.

File: package.json [hunk 5]
```diff
@@ -104,23 +104,24 @@
     "ember-cli-sass": "10.0.1",
     "ember-cli-server-variables": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-uglify": "^3.0.0",
+    "ember-cli-terser": "^4.0.0",
+    "ember-could-get-used-to-this": "^1.0.1",
```
Comment (stopfstedt) [inline package.json:108]: should be moved to `dependencies`.

**Review by stopfstedt — CHANGES_REQUESTED** [review]

File: addon/classes/resolve-async-value.js (hunk 1 above)
Comment (jrjohnson) [reply addon/classes/resolve-async-value.js]: ✔️ nice catch!

File: addon/classes/async-process.js (hunk 2 above)
Comment (jrjohnson) [reply addon/classes/async-process.js]: ✔️

File: package.json (hunk 5 above)
Comment (stopfstedt) [reply package.json]: that seems to have done the trick.

**Review by stopfstedt — APPROVED** [review]

**Pull request merged**